packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -160,6 +210,20 @@ class IMobileDevice {
       );
       if (result.exitCode == 255 && result.stdout != null && result.stdout.contains('No device found'))
         throw IOSDeviceNotFoundError('ideviceinfo could not find device:\n${result.stdout}');
+      if (result.exitCode == 255 && result.stderr != null && result.stderr.contains('Could not connect to lockdownd')) {
+        if (result.stderr.contains('error code -${LockdownReturnCode.pairingDialogResponsePending.code}')) {
+          throw const IOSDeviceNotTrustedError(
+            'Device info unavailable. Is the device asking to "Trust This Computer?" Try unlocking attached devices.',
```

Comment:
Sorry, with "Try unlocking attached devices" comment I meant to put it in the "ideviceinfo could not find device" error message above because the locked attached device's mux doesn't show up (they look like dumb USB devices like a keyboard).

Reply:
Hm, or maybe it would show up as LOCKDOWN_E_MUX_ERROR =  -8.  I honestly haven't tested it.
What you had before is fine.

Reply:
Updated